### PR TITLE
Refine README.md

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +30,57 @@ google spreadsheet add-on.
...
 ```
 
-### Testing
+## Structure
+
+The structure of the project consists of the following files:
+
+### Queries
+
+They can be found in the ``src/api/queries`` folder. All the queries, sent to the API can be seen here. A quick note here, the getMetric query is more dynamic.
```
Comment (kalkov) [inline README.md:53]: > the getMetric query is more dynamic. 

what does more dynamic mean here?

File: README.md [hunk 2]
```diff
@@ -11,21 +30,57 @@ google spreadsheet add-on.
...
+
+Found in the ``src/functions`` folder. Here lie all of the functions, connecting the google sheet functions and the API queries.
+
+### Sheet Functions
+
+Found in the ``src/`` folder. There are 2 files, one under the name of ``sheetFunctions.js`` and the other one being ``generatedSheetFunctions.js``, the former having all the functions, using APIs different from ``getMetric``, the latter having those who do not. The second file is generated, using the ``getMetricGenerator.js``.
+
+### Generators
+
+All of the generators are found in the ``src/generators`` folder.
```
Comment (kalkov) [inline README.md:65]: Explain what is a generator.

File: README.md [hunk 3]
```diff
@@ -11,21 +30,57 @@ google spreadsheet add-on.
...
+
+File Template Generators:
```
Comment (kalkov) [inline README.md:67]: Do not list the generators here, we will surely forget to keep that list up to date.

File: README.md [hunk 4]
```diff
@@ -3,6 +3,25 @@
...
+  - [Installation](#installation)
+  - [Structure](#structure)
+    - [Queries](#queries)
+    - [Functions](#functions)
+    - [Sheet Functions](#sheet-functions)
+    - [Generators](#generators)
+  - [Running tests](#running-tests)
+  - [Running integration tests](#running-integration-tests)
+  - [Standart template tests](#standart-template-tests)
+  - [Docker tests](#docker-tests)
```
Comment (kalkov) [inline README.md:21]: These are not docker tests. Move to subsection of Running Tests. Call it - "Run tests in docker container"

File: README.md [hunk 5]
```diff
@@ -3,6 +3,25 @@
...
+  - [Development](#development)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Structure](#structure)
+    - [Queries](#queries)
+    - [Functions](#functions)
+    - [Sheet Functions](#sheet-functions)
+    - [Generators](#generators)
+  - [Running tests](#running-tests)
+  - [Running integration tests](#running-integration-tests)
```
Comment (kalkov) [inline README.md:19]: Move to a subsection under Running tests

File: README.md (hunk 1 above)
Comment (spiderjako) [reply README.md:53]: I'll remove that, don't really know what I was going for with this one 🤷‍♂️

**Review by kalkov — APPROVED** [review]

**Pull request merged**